lambdas/authorizationcode/src/main/java/uk/gov/di/ipv/cri/passport/authorizationcode/AuthorizationCodeHandler.java

```diff
@@ -154,15 +154,15 @@ private void validateDcsResponse(DcsResponse dcsResponse)
         }
     }
 
-    private PassportGpg45Score generateGpg45Score(DcsResponse dcsResponse) {
+    private Evidence generateGpg45Score(DcsResponse dcsResponse) {
```

Comment:
Minor thing here but why the change in class name? Seems a bit confusing now that we have "Evidence" & a "Gpg45Evidence".